contrib/templates/experimental/scala-job/databricks_template_schema.json

```diff
@@ -0,0 +1,40 @@
+{
+  "properties": {
+    "project_name": {
+      "type": "string",
+      "description": "Name of project. Do not add spaces",
```

Comment:
Please folllow the conventions from https://github.com/databricks/cli/blob/main/libs/template/templates/default-sql/databricks_template_schema.json#L7-L10